ga4gh/backend.py

```diff
@@ -339,6 +354,13 @@ def _topLevelObjectGenerator(self, request, idMap, idList):
                 nextPageToken = str(currentIndex)
             yield object_.toProtocolElement(), nextPageToken
 
+    def _singleObjectGenerator(self, datamodelObject):
+        """
+        Returns a generator suitable for a search method in which the
+        result set is a single object.
+        """
+        return [(datamodelObject.toProtocolElement(), None)]
```

Comment:
yeah, I wasn't concerned about semantics, more about clarity... if a comment says a method returns a generator and it doesn't, it's confusing